norse/benchmark/bindsnet_lif.py

```diff
@@ -28,9 +28,12 @@ def lif_feed_forward_benchmark(parameters: BenchmarkParameters):
         .to(parameters.device)
         .float()
     )
+    input_spikes.requires_grad = False
 
     input_data = {"Input": input_spikes}
     network.to(parameters.device)
+    for param in network.parameters():
```

Comment:
Since we found that this actually makes performance worse, maybe omit?

Reply:
But omit I mean don't set requires_grad to false.

Reply:
Did we? I think we found that it improved the benchmark quite a bit... But I might be remembering wrongly?

Reply:
I thought the profiling traces for Norse showed that it made performance worse.

Reply:
Actually, yeah. It's pretty much the same:
![image](https://user-images.githubusercontent.com/1064317/94574002-f41cfd80-0272-11eb-9227-fec84a659cf5.png)
At some point it gave us a 0.5x boost. But this might have been solved with the JIT